qiskit/jobs/docker/1.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,144 @@
+
+# Install Open MPI
+RUN mkdir /tmp/openmpi \
+ && cd /tmp/openmpi \
+ && curl -fSsL -O https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.4.tar.gz \
+ && tar zxf openmpi-4.0.4.tar.gz \
+ && cd openmpi-4.0.4 \
+ && ./configure --enable-orterun-prefix-by-default \
+ && make -j $(nproc) all \
+ && make install \
+ && ldconfig \
+ && rm -rf /tmp/openmpi
+
+# Create a wrapper for OpenMPI to allow running as root by default
+RUN mv /usr/local/bin/mpirun /usr/local/bin/mpirun.real \
+ && echo '#!/bin/bash' > /usr/local/bin/mpirun \
+ && echo 'mpirun.real --allow-run-as-root "$@"' >> /usr/local/bin/mpirun \
+ && chmod a+x /usr/local/bin/mpirun
+
+RUN echo "hwloc_base_binding_policy = none" >> /usr/local/etc/openmpi-mca-params.conf \
+ && echo "rmaps_base_mapping_policy = slot" >> /usr/local/etc/openmpi-mca-params.conf
+
+ENV LD_LIBRARY_PATH=/usr/local/openmpi/lib:/usr/local/lib:$LD_LIBRARY_PATH
+ENV PATH /usr/local/openmpi/bin/:$PATH
+
+# SSH login fix. Otherwise user is kicked off after login
+RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+# Create SSH key.
+RUN mkdir -p /root/.ssh/ \
+ && mkdir -p /var/run/sshd \
+ && ssh-keygen -q -t rsa -N '' -f /root/.ssh/id_rsa \
+ && cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys \
+ && printf "Host *\n  StrictHostKeyChecking no\n" >> /root/.ssh/config
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    libbz2-dev \
+    libc6-dev \
+    libffi-dev \
+    libgdbm-dev \
+    liblzma-dev \
+    libncursesw5-dev \
+    libreadline-gplv2-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    tk-dev \
+ && rm -rf /var/lib/apt/lists/* \
+ && apt-get clean
+
+# Installing Python3
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
+ && tar -xvf Python-$PYTHON_VERSION.tgz \
+ && cd Python-$PYTHON_VERSION \
+ && ./configure && make && make install \
+ && rm -rf ../Python-$PYTHON_VERSION*
+
+# Upgrading pip and creating symbolic link for python3
+RUN ${PIP} --no-cache-dir install --upgrade pip setuptools
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
+ && ln -s $(which ${PIP}) /usr/bin/pip
+
+WORKDIR /
+
+RUN apt-get update && apt-get -y install cmake protobuf-compiler
+
+# Installing our custom libraries
+RUN ${PIP} install --no-cache --upgrade \
+        amazon-braket-default-simulator==1.3.0 \
+        amazon-braket-ocean-plugin==1.0.6 \
+        amazon-braket-schemas==1.8.0 \
+        amazon-braket-sdk==1.18.0 \
+        awscli==1.22.15 \
+        botocore==1.23.15 \
+        boto3==1.20.15 \
+        decorator==4.4.0 \
+        dask==2.30.0 \
+        dwave-ocean-sdk==4.2.0 \
+        matplotlib==3.2.2 \
+        numpy==1.21.4 \
+        pandas==1.1.4 \
+        rsa==4.4.1 \
+        scikit-learn==0.20.3 \
+        requests==2.26.0 \
+        scipy==1.7.3
+
+RUN ${PIP} install --no-cache --upgrade git+https://github.com/qiskit-community/qiskit-braket-plugin
```

Comment:
Due to the rename change to 

`RUN ${PIP} install --no-cache --upgrade git+https://github.com/qiskit-community/qiskit-braket-provider`

Reply:
installing from PyPi now :)